charts/csi-isilon/values.yaml

```diff
@@ -24,13 +24,13 @@ images:
 
   # CSM sidecars
   replication:
-    image: quay.io/dell/container-storage-modules/dell-csi-replicator:v1.10.0
+    image: quay.io/dell/container-storage-modules/dell-csi-replicator:v1.11.0
   podmon:
-    image: quay.io/dell/container-storage-modules/podmon:v1.11.0
+    image: quay.io/dell/container-storage-modules/podmon:v1.12.0
   authorization:
-    image: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0
+    image: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.13.0
```

Comment:
Authorization sidecar images can move to v2.1.0

Reply:
Done